docs/source/testing/trip_test.rst

```diff
@@ -0,0 +1,99 @@
+::
+    ./trip_test_command_line 155661f ec29bf3 -e 'axisymmetric' 'bucket_model' 'frierson' -r git@github.com:sit23/Isca
+
+**Please note that by default the trip tests will try to connect to the ExeClim/Isca repository using SSH**. If you use HTTPs please use the option ``-r https://github.com/ExeClim/Isca.git``.
+
+Example output
+--------------
+
+Running the command
+::
+    ./trip_test_command_line 155661f8c7945049cbac0dcf2019bb17fe7a6a8d ec29bf389cf5ac53b50b23c363040479a6392e52
+
+Produces the following summary output
+::
+    Results for all of the test cases ran comparing 155661f and ec29bf3 are as follows...
+    variable_co2_grey : pass
+    realistic_continents_fixed_sst : pass
+    realistic_continents_variable_qflux : pass
+    held_suarez : pass
+    variable_co2_rrtm : pass
+    bucket_model : fail
+    top_down_test : pass
+    frierson : pass
+    axisymmetric : pass
+    MiMA : pass
+    giant_planet : pass
+    Nightmare, some tests have failed
+
+The bucket test is the only one that fails in this instance. This is because the bucket model formulation was changed between these two commits, and so it is expected that the results with the two commits will differ. However, we note that the other tests have their results unchanged, meaning this is a safe modification to the code. Any unexpected failures should be investigated before submitting a pull-request.
+
+FAQs
+----
+
+* **What happens if the trip tests fail for my modifications?** If some of the trip tests fail, please investigate why you think this might be. If the tests fail on an isolated test case that you haven't used before, please post the results in your PR and we can discuss what is best to do. We will accept some PRs that fail the trip tests, but we will need to understand why first.
+
+* **When I submit a pull request, which two commit IDs should I use?** Please use the commit ID of the current HEAD of Isca's master branch as one, and the HEAD of your pull-requested branch as the other. *Please note that in order for this to work, you will have to use the* `-r` *option to select your own fork, as described above.*
+
+* **How long do the trip tests take to run?** The code is re-compiled several times during the trip tests, so the majority of time will be spent compiling. Each test case is only run for a few days, so once the code has compiled it shouldn't take more than 30 mins or so, depending on how many cores you're using. You can increase the number of cores by using the ``-n`` option.
+
+* **I am getting an error message saying that git cannot find my commit ID - how do I fix this?** If you are specifying commit IDs that are present on your fork but not in the ExeClim version of Isca, you will need to specify your own fork as the repository. To do this, use the ``-r`` option, e.g. ``-r git@github.com:sit23/Isca``. Note that you can specify the repository location using the HTTPS option, or the SSH option depending on your preference.
```

Comment:
for _sit23_ suggest you use _usr_  (as well as being more accurate it will avoid people running your code base instead of theirs - ie it will fail)